src/main/java/com/minecolonies/coremod/colony/ColonyList.java

```diff
@@ -15,47 +15,27 @@
 import java.util.stream.StreamSupport;
 
 //TODO extend list/collection
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)